RELEASE_NOTES.md

```diff
@@ -1,29 +1,37 @@
-<!--
-  A release notes template that should be adapted for every release
-    - release: <v*.*.*>
-    - release branch: <v*.x>
-    - the last release: <v-last> 
-    - the last release branch: <v-last.x>
--->
-
-# Gaia <v*.*.*>  Release Notes 
+# Gaia v21.0.0 Release Notes 
 
 ## 📝 Changelog
 
-Check out the [changelog](https://github.com/cosmos/gaia/blob/<v*.*.*>/CHANGELOG.md) for a list of relevant changes or [compare all changes](https://github.com/cosmos/gaia/compare/<v-last>...<v*.*.*>) from last release.
+Check out the [changelog](https://github.com/cosmos/gaia/blob/v21.0.0/CHANGELOG.md) for a list of relevant changes or [compare all changes](https://github.com/cosmos/gaia/compare/v20.0.1....v21.0.0) from last release.
 
 <!-- Add the following line for major releases -->
-Refer to the [upgrading guide](https://github.com/cosmos/gaia/blob/release/<v*.x>/UPGRADING.md) when migrating from `<v-last.x>` to `<v*.x>`.
+Refer to the [upgrading guide](https://github.com/cosmos/gaia/blob/release/v21.x/UPGRADING.md) when migrating from `v20.0.x` to `v21.x`.
 
 ## 🚀 Highlights
 
 <!-- Add any highlights of this release -->
 
+This release bumps Interchain Security (ICS) to [v6.2.0](https://github.com/cosmos/interchain-security/releases/tag/v6.2.0) which brings the following improvements:
+
+- Populate the memo on the IBC transfer packets used to send ICS rewards
+with the required consumer chain Id to identify the consumer to the provider, see [#2290](https://github.com/cosmos/interchain-security/pull/2290).
+
+- Enable permissionless allowlisting of reward denoms (at most 3) per consumer chain, see [#2309](https://github.com/cosmos/interchain-security/pull/2309) for more details.
+
+It also bumps CosmWasm/wasmvm to [v2.1.3](https://github.com/CosmWasm/wasmvm/releases/tag/v2.1.3)
+
+Two minor bugs in the invariant submission and the creation of multiple temporary directories have also been fixed in this release.
+
+The release also distributes all the unaccounted known denoms from  the
+ consumer rewards pool via migration.
```

Comment:
```suggestion

The release also distributes all the unaccounted known denoms from  the consumer rewards pool via migration.
```
Just keep highlights in the release notes. The changelog already contains all changes.